src/ghdcbot/adapters/github/rest.py

```
@@ -526,6 +529,226 @@ def create_issue_comment(
+        repo: str,
+        template_name: str,
+    ) -> str | None:
+        """Fetch a Markdown GitHub issue template from .github/ISSUE_TEMPLATE/{name}.md.
+
+        Returns the raw Markdown file content (with YAML frontmatter still
+        included — caller should strip it) or None if not found or if the template
+        is not a Markdown file.
+        """
+        import base64
+
+        clean_name = template_name[:-3] if template_name.endswith(".md") else template_name
```

Ruff (0.16.3)

[error] Prefer `str.removesuffix()` over conditionally replacing with slice.

Use removesuffix instead of ternary expression conditional upon endswith.

(FURB188)